modules/ROOT/pages/_partials/generic-binding.adoc

```diff
@@ -0,0 +1,20 @@
+include::{partialsdir}/attributes.adoc[]
+
+After binding a {mobile-client} to the {service-name} service, your mobile app can use that service immediately. 
+
+: The association between a {mobile-client} and {service-name} service uses the xref:https://blog.openshift.com/asynchronous-bind-with-the-automation-broker/[OpenShift bind] feature.
+
+To associate a {mobile-client} with a mobile service:
+
+. Navigate to the *Overview* your OpenShift project.
+
+. Expand the {mobile-client} you want to associate by clicking the *>* icon.
++
+NOTE: Your mobile app will be represented by two items in OpenShift, an item in the *{mobile-client}s* list and an item in the *Provisioned Services* list. Make sure to expand the item in the *{mobile-client}s* list.
+
+. Note the value for `clientId` in the *Config Info* field. You will need this for the next step.
+
+. Choose the {service-name} service from the *Integrations* list. If an option is not available, you can provision that service by choosing that option and associate the {mobile-client} later.
+
+. Follow the *Create Binding* wizard to associate the {mobile-client} with the {service-name} service. You may be prompted for a *{mobile-client} ID*. Enter the value you noted from the previous step.
```

Comment:
"You may be prompted for a *{mobile-client} ID*. Enter the value you noted from the previous step." --> If prompted for id...